### PR TITLE
Update vtunnel documents

### PR DESCRIPTION
**Pull request by Nino-K**
Related to: https://github.com/rancher-sandbox/rancher-desktop/issues/2327

Signed-off-by: Nino Kodabande <nkodabande@suse.com>

### CODE REVIEW

File: src/go/vtunnel/README.md [hunk 1]
```diff
@@ -40,17 +40,27 @@ You can simply run the e2e test:
  GOOS=windows go build
  GOOS=linux go build
 ```
+- Creat a configuration file, below is an example a config.yaml.
```
Comment (mook-as) [inline src/go/vtunnel/README.md:43]:
```suggestion
- Creat a configuration file, below is an example a `config.yaml`.
```

File: src/go/vtunnel/README.md [hunk 2]
```diff
@@ -40,17 +40,27 @@ You can simply run the e2e test:
...
+  - handshakePort: 9090
+    vsockHostPort: 8989
+    peerAddress: 127.0.0.1
+    peerPort: 3030
+    upstreamServerAddress: 127.0.0.1:4444
+ ```
  - Move the vtunnel to the hyper-v VM and run the Peer process:
  ```bash
- ./vtunnel peer --handshake-port 9090 --vsock-port 8989 --listen-address 127.0.0.1:3030
+ ./vtunnel peer --configPath config.yaml
```
Comment (mook-as) [inline src/go/vtunnel/README.md:55]: Interesting; I'd have expected `--config-path` instead.

File: src/go/vtunnel/README.md [hunk 3]
```diff
@@ -40,17 +40,27 @@ You can simply run the e2e test:
...
+ **Note** the configuration for both Peer and Host process is identical.
+ ```yaml
+ tunnel:
+  - handshakePort: 9090
+    vsockHostPort: 8989
+    peerAddress: 127.0.0.1
+    peerPort: 3030
+    upstreamServerAddress: 127.0.0.1:4444
+ ```
  - Move the vtunnel to the hyper-v VM and run the Peer process:
```
Comment (mook-as) [inline src/go/vtunnel/README.md:53]:
```suggestion
 - Move the `vtunnel` executable to the Hyper-V VM and run the Peer process:
```

**Review by mook-as — APPROVED** [review]

File: src/go/vtunnel/README.md (hunk 2 above)
Comment (Nino-K) [reply src/go/vtunnel/README.md:55]: @mook-as this was `--config-path` before, but you [suggested](https://github.com/rancher-sandbox/rancher-desktop/pull/2436#discussion_r902949910) changing it to `--configPath` 😹

Comment (mook-as) [reply src/go/vtunnel/README.md:55]: 🤣 in that comment I meant to name the _variable_ (the one being assigned to) `configPath`, not the parameter; my bad for having a terrible review comment.

This isn't going to block merging this PR.

**Pull request merged**